stylus/index.js

```diff
@@ -12,13 +12,16 @@
 var path = require('path')
 var stylus = require('stylus')
 
+const NODE_MODULES_PATH = path.resolve(__dirname, '../..')
```

Comment:
Isn't it ` const NODE_MODULES_PATH = path.resolve(__dirname, '../node_modules')` ?

Reply:
It depends on the directory this file is located.
In `cozy-labs/cozy-desktop`, `cozy-ui` is located in `node_modules` as it is a dependency. And so `cozy-ui/stylus/index.js` is located in `node_modules`. To retrieve the `node_modules` folder from this location, it is `node_modules/cozy-ui/stylus/../../`.